README.md

```diff
@@ -448,7 +482,7 @@ Microsoft Windows* Server | 2016, 2019 | *Not supported*
     <tbody>
         <td rowspan=5> Linux*/Windows* </td>
         <td rowspan=2> x86 CPU </td>
-        <td> Intel(R) oneAPI DPC++ Compiler <br> or <br> Intel project for LLVM* technology </td>
```

Comment:
If we're modifying this table, we should get rid of the *Installed by Conan* column. See https://github.com/oneapi-src/oneMKL/discussions/267

Reply:
Since Conan permeates README, I think it would be better to do a full clean-up later instead of spot cleaning a table now.

Reply:
OK!